alphashield/database/mongodb_client.py

```
@@ -1,132 +1,73 @@
+                        decision["timestamp"] = datetime.utcnow()
+                    DecisionDoc(**decision)
+                    db.decisions.update_one(
+                        {
+                            "agent_id": decision["agent_id"],
+                            "loan_id": decision["loan_id"],
+                            "timestamp": decision["timestamp"],
+                        },
+                        {"$setOnInsert": decision},
+                        upsert=True,
+                    )
+                except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)